docs/src/design/aux_table/pow2.md

```diff
@@ -15,7 +15,8 @@ The columns shown above have the following meanings:
 - **Helper column $h$** is used to help aggregate the powers of two into the output column and to ensure correctness of the transition of $a_i$ values from one row to the next.
 - **Input column $a$** will contain the aggregated sum of the $a_i$ values in each row plus the previous value in column $a$. By the final row of the 8-row cycle, the value in column $a$ will be the input value of the exponent $a$.
 - **Column $p$** will contain increasing powers of 256, which will be used in the aggregation of the output result in column $z$. This represents the 8 powers of two that can be accumulated in each row by the 8 $a_i$ columns.
-- **Output column $z$** will contain the aggregated output over each row plus the previous value of $z$. By the final row of the 8-row cycle, the value in column $z$ will be the result of $2^a$.
+- **Output column $zp$** will contain the aggregated output over all rows excluding the current row.
+- **Output column $z$** will contain the aggregated output over each row plus the previous value of $z$ i.e. current value of $zp$. By the final row of the 8-row cycle, the value in column $z$ will be the result of $2^a$.
```

Comment:
I don't think this is quite correct the way it is stated. It actually contains the aggregated output over the current row plus the previous value of $z$. I'd prefer: "**Output column $z$** will contain the aggregated output over the current row plus the previous value of $z$ i.e. the current value of $zp$. By the final row of the 8-row cycle, the value in column $z$ will be the result of $2^a$."

Reply:
Fixed! Please see [bfa8a00](https://github.com/maticnetwork/miden/pull/232/commits/bfa8a0019bce0ea03c28f1d1f5e83672e00b4066).